app/assets/stylesheets/screen.scss

```diff
@@ -319,60 +293,43 @@ div.story-title abbr.initials {
   cursor: inherit;
 }
 
-span.estimate {
-  border: 1px solid #777;
-  padding: 0 0.2em;
-  margin: 0 2px 0 0;
-  @include border-radius(3px);
-  vertical-align: top;
-  font-size: x-small;
-  font-weight: bold;
-  font-style: italic;
+.estimate {
+  border: none;
+  padding: 0;
+  margin: 0;
+  font-size: 10px;
+  transform: rotate(90deg);
+  display: inline-block;
+  letter-spacing: -1px;
+  width: 20px;
+  margin-left: -2px;
+  margin-right: -6px;
+  text-align: right;
+  top: -7px;
+  position: relative;
+  color: #3556AA;
+  height: 17px;
+  -webkit-font-smoothing: none;
+  cursor: pointer;
 }
+.estimates {
+  padding-left: 10px;
+  padding-right: 2px;
 
-span.estimate_1 {
-  @include background(linear-gradient(bottom,
-      $orange 12.5%,
-      white 12.5%));
-}
-span.estimate_2 {
-  @include background(linear-gradient(bottom,
-      $orange 25%,
-      white 25%));
-}
-span.estimate_3 {
-  @include background(linear-gradient(bottom,
-      $orange 37.5%,
-      white 37.5%));
-}
-span.estimate_4 {
-  @include background(linear-gradient(bottom,
-      $orange 50%,
-      white 50%));
-}
-span.estimate_5 {
-  @include background(linear-gradient(bottom,
-      $orange 62.5%,
-      white 62.5%));
-}
-span.estimate_6 {
-  @include background(linear-gradient(bottom,
-      $orange 75%,
-      white 75%));
-}
-span.estimate_7 {
-  @include background(linear-gradient(bottom,
-      $orange 87.5%,
-      white 87.5%));
-}
-span.estimate_8 {
-  @include background(linear-gradient(bottom,
-      $orange 100%,
-      white 100%));
+  .estimate {
+    font-weight: normal;
+    border-radius: none;
+    margin-left: -3px;
+    border-right: 1px solid #858484;
+    padding-right: 2px;
+    top: -3px;
+    background: transparent;
+    line-height: 15px;
```

Comment:
[It's a good practice not to use units when defining a value for the `line-height` property](https://css-tricks.com/almanac/properties/l/line-height/#article-header-id-0), it can lead to unexpected behavior with child elements (although I think won't be the case here).